src/integration/java/io/pinecone/integration/controlPlane/pod/CreateDescribeListAndDeleteIndexTest.java

```diff
@@ -69,6 +69,44 @@ public void createIndexWithPodsAndPodType() {
         controlPlaneClient.deleteIndex(podIndexName);
     }
 
+    @Test
+    public void createPodsIndexWithMinimumRequiredParams() {
+        String podIndexName = RandomStringBuilder.build("create-pod-with-min", 8);
+        Integer dimension = 3;
+        String environment = "us-east-1-aws";
+        String podType = "p1.x1";
+        IndexModel podsIndex = controlPlaneClient.createPodsIndex(podIndexName, dimension, environment, podType);
+
+        assertEquals(podIndexName, podsIndex.getName());
+        assertEquals(dimension, podsIndex.getDimension());
+        assertEquals(environment, podsIndex.getSpec().getPod().getEnvironment());
+        assertEquals(podType, podsIndex.getSpec().getPod().getPodType());
+
+        // Confirm defaults are put in by the backend when not supplied by the user
+        assertEquals(IndexMetric.COSINE, podsIndex.getMetric());
+        assertEquals(1, podsIndex.getSpec().getPod().getPods());
+        assertEquals(1, podsIndex.getSpec().getPod().getReplicas());
+        assertEquals(1, podsIndex.getSpec().getPod().getShards());
+        assertNull(podsIndex.getSpec().getPod().getMetadataConfig());
+        assertNull(podsIndex.getSpec().getPod().getSourceCollection());
+
+        // Cleanup
+        controlPlaneClient.deleteIndex(podIndexName);
+    }
+
+    @Test
+    public void CreatePodsIndexWithInvalidIndexName() {
+        String podIndexName = "Invalid-name";
+        Integer dimension = 3;
+        String environment = "us-east-1-aws";
+        String podType = "p1.x1";
+
+        // Empty indexName
+        assertThrows(PineconeValidationException.class, () -> {
```

Comment:
Only put this here, since I didn't want to duplicate all of the unit test logic in the integration tests. LMK thoughts on whether I should test more logic here (e.g. negative dimension passed).